modules/rest-api/pages/backup-node-threads.adoc

```diff
@@ -0,0 +1,181 @@
+= Manage Backup Service Threads
+:description: You can change the number of threads a Backup Service node uses when backing up data.
+:stem: latexmath
+
+[abstract]
+{description}
+
+== HTTP Methods and URIs
+
+.Get all overrides to the thread Backup Service settings:
+[source, uri]
+----
+GET /api/v1/nodesThreadsMap
+----
+
+.Overwrite all thread settings with new values:
+[source, uri]
+----
+POST /api/v1/nodesThreadsMap
+----
+
+.Update some thread settings:
+[source, uri]
+----
+PATCH /api/v1/nodesThreadsMap
+----
+
+== Description
+
+The `nodesThreadMap` endpoint lets you change the number of threads the Backup Service uses.
+The default number of threads the Backup Service uses is based on the number of CPU cores in the node: stem:[\max(1, cpu\_cores \times 0.75)].
+The number of threads also sets the number of concurrent client connections the service uses to retrieve data from nodes in the cluster.
+Each thread creates one connection.
+See xref:learn:services-and-indexes/services/backup-service.adoc#threads[Thread Usage] for more information about how the number of threads affects the Backup Service.
+
+== Curl Syntax
+
+.Get the current thread overrides
+[source, console]
+----
+curl -u $USER:$PASSWORD -X GET \
+     http://$BACKUP_SERVICE_NODE:$BACKUP_SERVICE_PORT/api/v1/nodesThreads
+----
+
+.Overwrite all thread settings
+[source, console]
+----
+ curl -u Administrator:password -X POST \
+      http://$BACKUP_SERVICE_NODE:$BACKUP_SERVICE_PORT/api/v1/nodesThreadsMap \
+      -d <nodes_thread_map>
+----
+
+.Update/set some thread settings
+[source, console]
+----
+ curl -u Administrator:password -X PATCH \
+      http://$BACKUP_SERVICE_NODE:$BACKUP_SERVICE_PORT/api/v1/nodesThreadsMap \
+      -d <nodes_thread_map>
+----
+
+.POST Parameters
+[cols="2,3,2"]
+|===
+|Name | Description | Schema
+
+| `nodes_thread_map`
+| An object that sets the number of threads for Backup Service nodes. 
+When you use the `PATCH` method, the changes only apply to the nodes in the map.
+Any existing settings that are not in the map remain in effect.
+Calling `POST`, removes any existing overrides.
+After the call, only the overrides you supply in the map are in effect.
+| <<nodes_thread_map_schema,Node Threads Map>>
+
+|===
+
+[#nodes_thread_map_schema]
+.Node Thread Map Schema
+[source, json]
+----
+{"nodes_threads_map": {
+    <backup_node_uuid>:<threads>, . . .  
+    }
+}
+----
+
+[cols="2,3,2"]
+|===
+|Name | Description | Schema
+
+| `backup_node_uuid`
+| The unique identifier for a node running the Backup Service. 
+You can get this value from the `/pools/nodes` REST API. 
+See xref:rest-api:rest-node-get-info.adoc[].
+| string
+
+| `threads`
+| The number of threads for the Backup Service to use.
```

Comment:
Please add info on what the value "0" means -- for example:
If you set the value to 0, it means that the default behavior of the Backup Service for the number of threads (as explained in the [description](https://preview.docs-test.couchbase.com/cbbackupmgr-threads/server/current/rest-api/backup-node-threads.html#description)) should be used.

Reply:
Good catch. Added.